ui/test/nuxeo-document-comment.test.js

```diff
@@ -0,0 +1,450 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import '../nuxeo-document-comments/nuxeo-document-comment.js';
+import {
+  fixture,
+  flush,
+  html,
+  isElementVisible,
+  timePasses,
+  waitForAttrMutation,
+  waitForEvent,
+} from '@nuxeo/nuxeo-elements/test/test-helpers';
+import { pressAndReleaseKeyOn, tap } from '@polymer/iron-test-helpers/mock-interactions';
+import fakeServer from '@nuxeo/nuxeo-elements/test/nuxeo-mock-client';
+
+function getCommentContent(element) {
+  return element.shadowRoot.querySelector('#content');
+}
+
+/* eslint-disable no-unused-expressions */
+suite('<nuxeo-document-comment>', () => {
```

Comment:
I though this was an implementation detail of `<nuxeo-document-comment-thread>` and would be tested as part of the tests on that element, no ?

Reply:
Not really! It is a detail but it could also be seen as a separate element from a functional point of view.
Anyway a question was created to product team in order to review comments' elements in general (e.g. comment's input may be a candidate for being extracted).